src/components/Help/Stepper.scss

```diff
@@ -0,0 +1,14 @@
+.box {

```

Comment:
This should be a more descriptive class name.

Reply:
renamed gifContainer